.github/workflows/ci.yml

```diff
@@ -45,6 +45,7 @@ jobs:
           - ember-lts-3.16
           - ember-lts-3.20
           - ember-lts-3.24
+          - ember-lts-3.28
```

Comment:
Where's the Ember 4 here in the package.json?

Reply:
ember-canary, release and beta channels are the ember 4.
I'm not sure whats the usual practice now for testing ember 4 compatibility, but preferably we should test only against LTS versions when there is no v4 LTS yet?

Reply:
Ya, I generally agree. But since Ember does publish alphas from master (and master is 4.4.x), we can add `ember-lts-4.4` and use `^4.4.0-alpha.1` in ember-try config.

Reply:
Thanks, feedback applied.